Tests/RoutingKitTests/RouterTests.swift

```diff
@@ -106,16 +106,33 @@ class RouterTests: XCTestCase {
         print(params)
     }
     
-    func testPerformance() throws {
+    func testCaseSensitivePerformance() throws {
+        let router = TrieRouter(String.self)
+        for letter in ["a", "b", "c", "d", "e" , "f", "g"] {
+            router.register(route: Route(path: [
+                .constant(letter),
+                .parameter("\(letter)_id")
+            ], output: letter))
+        }
+       
         measure {
```

Comment:
I've moved the setup outside the performance measurement. Chances are it wasn't impacting the tests much, but since we want to frontload work during route registration, and we only really care how fast the actual routing is, it makes sense to make those facts explicit in our tests.